src/cmd/check.go

```diff
@@ -54,22 +56,74 @@ var listCheckCmd = &cobra.Command{
 var checkCreateCmd = &cobra.Command{
 	Use:   "check",
 	Short: "Create a rubric check",
-	Long: `Create a rubric check
-
-Examples:
-
-	opslevel create check -f my_cec.yaml
-`,
+	Long:  `Create a rubric check`,
+	Example: `
+cat << EOF | opslevel create check -f -
+Version: "1"
+kind: "repo_grep"
+spec:
+  name: "new repo grep check"
+  enabled: false
+  category: "misc"
+  level: "silver"
+  notes: "some notes"
+  directorySearch: false
+  filePaths:
+  - "Taskfile.yml"
+  fileContentsPredicate:
+    type: "exists"
+    value: ""
+EOF`,
 	Run: func(cmd *cobra.Command, args []string) {
-		usePrompts := !hasStdin()
-		input, err := readCheckCreateInput()
+		input, err := readCheckInput()
 		cobra.CheckErr(err)
+		usePrompts := true
+		if cmd.Flag("noninteractive").Changed {
+			usePrompts = false
+		} else if dataFile == "-" {
```

Comment:
Using a heredoc without disabling prompt UI results in this error message, because the input is sent to the prompt. Here's the error message I was getting without this `else if`:

```
Error: ^D
exit status 1
```

Reply:
ahhh neat!